src/Components/User/WorkshopLogin.js

```diff
@@ -6,16 +6,19 @@ import Col from "react-bootstrap/Col";
 import FormControl from "react-bootstrap/FormControl";
 import Container from "react-bootstrap/Container";
 import NavBar from "../Layout/NavBar";
+import Alert from 'react-bootstrap/Alert'
```

Comment:
Refer to #33 for reducing import clutter

Reply:
Change has been implemented.